docs/modules/operation/pages/deep-dive/service-assurance/scheduled-outages.adoc

```diff
@@ -0,0 +1,91 @@
+
+= Scheduled Outages
+
+Scheduled outages let you suppress notifications and pause status polling, threshold checking, and data collection during a specified time period.
+You might want to schedule an outage for any of the following reasons:
+
+* Updating components.
+* Completing scheduled reboots.
+* Planned decommissioning of nodes.
+* Preventing excessive notifications to on-call staff.
+* Completing scheduled infrastructure testing.
+* Other situations where thresholds may be reached or outages may be caused.
+
+To configure scheduled outages in {page-component-title}, click the *gear* symbol at the top-right of the page and, under Service Monitoring, select *Configure Scheduled Outages*.
+
+== Create a scheduled outage
```

Comment:
```suggestion

```
Remove this so we don't have two of the same topics on the same page. And moving the content as Emily suggests should work.